barretenberg/cpp/src/barretenberg/commitment_schemes/gemini/gemini.hpp

```diff
@@ -183,42 +172,77 @@ template <typename Curve> class GeminiVerifier_ {
         return fold_polynomial_opening_claims;
     }
 
-  private:
+    static std::vector<Commitment> get_gemini_commitments(size_t log_circuit_size, auto& transcript)
```

Comment:
`log_circuit_size` is const here and below, so is `commitment` in th body of this function